setup.cfg

```diff
@@ -140,8 +141,10 @@ transformers =
     peft
 vllm =
     vllm>=0.2.6
+    flashinfer @ https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.1/flashinfer-0.1.1+cu121torch2.3-cp311-cp311-linux_x86_64.whl  ; sys_platform=='linux'  #for gemma 2 which uses Fashinfer for inference
 sglang =
-    sglang[all]
+    sglang>=0.1.22 ; sys_platform=='linux'
+    flashinfer @ https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.1/flashinfer-0.1.1+cu121torch2.3-cp311-cp311-linux_x86_64.whl  ; sys_platform=='linux'  #for gemma 2 which uses Fashinfer for inference
```

Comment:
You cannot limit the package version here. Why do users have to use `cp311` and `cu121`?